hadoop-ozone/dist/src/main/compose/ozone/test-ranger.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#suite:misc
```

Comment:
Adding the script to the `failing` suite makes CI skip it.  This way we can commit the patch before amd64 images are available, and let developers using Mac try it locally.

```suggestion
#suite:failing
```

Alternatively we can rename the script to `disabled-test-ranger.sh`.

Reply:
Sure, makes sense.